src/Microsoft.HttpRepl.Fakes/MockHttpMessageHandler.cs

```diff
@@ -9,16 +11,46 @@ namespace Microsoft.HttpRepl.Fakes
 {
     public class MockHttpMessageHandler : HttpMessageHandler
     {
-        private HttpResponseMessage response;
+        private string _fileContents;
```

Comment:
Should all of these be `readonly`?

Reply:
Yes, updated here:
https://github.com/aspnet/HttpRepl/commit/6b03992aef4b9c8a1c2eebec091249e5b3e1069d

Thanks!